### DISCUSSION TITLE
Monitor speakers sync issue

### DISCUSSION THREAD

**Discussion by PoeticMichael** [MonitorControl/MonitorControl#1668] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

It's not exactly MonitorControl problem, but maybe you can solve it. 
The problem happens with two different USB-C-connected monitors - Samsung and LG.
It shows up both with Sonoma running MacBook Pro and Monterey MacBook Pro. M1 and M2.
With MonitorControl and without MonitorControl.

If the laptop reboots while not connected to the monitor, sound through the monitor speakers will be garbled. 
Videos that have sound are reproduced in a very jittery way, skipping fragments, scanning back and forth, etc.

If I reboot the laptops with the monitors connected, there is no issue, even if the machine is disconnected and reconnected later.

### Describe the solution you'd like

It looks like there is some synchronization with monitors happening at reboot and at reboot only.
Is there a chance MonitorControl could do some sync to avoid this issue?

### Describe alternatives you've considered

Reboot while connected to the monitor is the only solution I found.

**Comment by waydabber** [comment]
This seems to be some kind of hardware issue. You can try using BetterDisplay and use Tools/Displays.../Reinitialize External Displays to check whether that helps.

**Comment by PoeticMichael** [comment]
Interesting. I installed BetterDisplay, and after reboot, the issue hasn't reproduced. 
I will give it some mileage and report back here with any findings. Googling around, I see that this is a pretty prevalent problem with in-monitor speakers, USB-C and MacOS.
